.travis.yml

```diff
@@ -50,12 +50,13 @@ install:
   - python run.py -t &
 
 before_script:
-  - "sleep 2"
+  - sleep 5
   # lint JS in default eslint task
   - npm run lint
   - npm run build
 
 # now run the tests!
 script:
   - nosetests
+  - sleep 5
```

Comment:
* I searched for similar issues online and one comment was saying that 10s was not enough and they put 20s. 
* Also most of the starting script for servers have `python run.py > /dev/null &`

Reply:
nosetests phase doesn't need the python server to be started.